python-lib/ontology_tagger.py

```diff
@@ -205,17 +201,21 @@ def tag_and_format(
         -Split sentences by applying sentencizer on documents
         -Use the right Matcher depending on the presence of categories
         """
-        tokenizer = MultilingualTokenizer(use_models=True, split_sentences=True)
-        logging.info(f"Splitting sentences on {len(text_df)} documents...")
-        start = perf_counter()
+        tokenizer = MultilingualTokenizer(
+            use_models=True, split_sentences=True, enabled_components=["sentencizer"]
```

Comment:
Could we remove the split_sentences parameter, since we already have that information in enable_components?

Reply:
I changed Multilingual Tokenizer arguments according to our previous discussion, in here : f6c6121